api/types.go

```diff
@@ -222,7 +223,7 @@ type EmbeddingRequest struct {
 
 // EmbeddingResponse is the response from [Client.Embeddings].
 type EmbeddingResponse struct {
-	Embedding []float64 `json:"embedding"`
+	Embedding [][]float64 `json:"embedding"`
```

Comment:
This looks to be a breaking change. Single string prompts should still return a `[]float64` not a `[][]float64`.

This can be a separate change but we should probably be returning a `float32` here...